client/my-sites/pages/page-list.jsx

```diff
@@ -379,6 +399,60 @@ const mapState = ( state, { query, siteId } ) => ( {
 	pages: getPostsForQueryIgnoringPage( state, siteId, query ) || [],
 	site: getSite( state, siteId ),
 	newPageLink: getEditorUrl( state, siteId, null, 'page' ),
+	homepageType: getSiteFrontPageType( state, siteId ),
 } );
 
-const ConnectedPages = flowRight( connect( mapState ), localize, withLocalizedMoment )( Pages );
+const ConnectedPages = flowRight(
```

Comment:
Move the codes below to the end of this file to fix eslint because `Pages` should be defined before use